packages/components/src/constants.ts

```diff
@@ -19,6 +19,8 @@ export const QUERY_GRID_PREFIX = 'labkey-querygrid-';
 export const FASTA_EXPORT_CONTROLLER = 'biologics';
 export const GENBANK_EXPORT_CONTROLLER = 'biologics';
 export const SAMPLE_SET_DISPLAY_TEXT = 'Sample Type';
+export const STICKY_HEADER_HEIGHT = 60; // Issue 38478
```

Comment:
Isn't this set in the SM app side and passed into the various components that need it? Just curious why it is needed to be defined on the shared component side?

Reply:
@labkey-ians

Reply:
I'm using this as the default